# Unidad 9 Ensamblaje de genomas

## Sesión 1 – Ensamblaje de novo de genomas completos

Alan Michael Torres Calderón - LANGEBIO

1. [Presentación de Ensamblado de genomas completos](Sesion1_ENSAMBLE_2019.pdf)

2. [Tutorial hands on](ensamblado_genomas_handson.md)

3. [Video de la clase](http://youtu.be/J5aWNiUZfEE)


## Sesión 2 - Ensamblaje de representación reducida de genomas

Alicia Mastretta

1. [Presentación de genomas reducidos (RAD, GBS, etc)](Sesion2_RAD_GBS_nFriends.pdf)

2. Tutoriales:

**Stacks**

Rochette, N. C. & Catchen, J. M. Deriving genotypes from RAD-seq short-read data using Stacks. Nature Protocols 12, 2640–2659 (2017)

[Demo scripts](https://bitbucket.org/rochette/rad-seq-genotyping-demo/src/default/)

[Demo data](http://catchenlab.life.illinois.edu/data/rochette2017_gac_or.tar.gz)  

**ipyrad**

[Introductory tutorial](https://ipyrad.readthedocs.io/tutorial_intro_cli.html#create-an-ipyrad-params-file)

[Advanced tutorial](https://ipyrad.readthedocs.io/tutorial_advanced_cli.html#tutorial-advanced-cli)

[Jupyter and more tutorials](https://ipyrad.readthedocs.io/userguide.html)

3. Videos:

[Video de la presentación de la clase](https://www.youtube.com/watch?v=AbmJSyoQLi0)

[Video de una charla de optimización de parámetros](https://www.youtube.com/watch?v=015pC1lD6mY#t=03h21m36s)

4. Referencias:

Revisión de los diferentes protocolos RAD:

[Andrews KR, Good JM, Miller MR, Luikart G, Hohenlohe PA (2016) **Harnessing the power of RADseq for ecological and evolutionary genomics**. Nature Reviews Genetics, 17, 81–92.](http://www.nature.com/nrg/journal/v17/n2/full/nrg.2015.28.html)

Software:

[Eaton, D. A. (2014). **PyRAD: assembly of de novo RADseq loci for phylogenetic analyses**. Bioinformatics, 30(13), 1844–1849.](https://academic.oup.com/bioinformatics/article/30/13/1844/2422183)

[Rochette, N. C. & Catchen, J. M. **Deriving genotypes from RAD-seq short-read data using Stacks**. Nature Protocols 12, 2640–2659 (2017)](www.nature.com/articles/nprot.2017.123)


Métodos optimización de parámetros *de novo*:

[Mastretta-Yanes A, Arrigo N, Alvarez N et al. (2015) **Restriction site-associated DNA sequencing, genotyping error estimation and de novo assembly optimization for population genetic inference.** Molecular Ecology Resources, 15, 28–41.](http://onlinelibrary.wiley.com/wol1/doi/10.1111/1755-0998.12291/full)

[Paris, J. R., Stevens, J. R., & Catchen, J. M. (2017). **Lost in parameter space: a road map for stacks**. Methods in Ecology and Evolution, 8(10), 1360–1373. doi: 10.1111/2041-210X.12775](https://onlinelibrary.wiley.com/doi/10.1111/2041-210X.12775/full)
